migrations/env.py

```diff
@@ -65,6 +65,19 @@ def process_revision_directives(context, revision, directives):
                 directives[:] = []
                 logger.info('No changes in schema detected.')
 
+    # this function is used to filter out the postgis index / spatial_ref_sys
+    # tables when identifying changes. these are not mentioned in our local
+    # model and as such the migration script sees them as "removed".
+    # reference: https://github.com/miguelgrinberg/Flask-Migrate/issues/18
+    # reference: http://alembic.zzzcomputing.com/en/latest/api/runtime.html
+    def include_object(object, name, type_, reflected, compare_to):
+        # two whitelisted elements: index `idx_destinations_point` and table
+        # `spatial_ref_sys`
+        return not any([
+            type_ == 'index' and name == 'idx_destinations_point',
```

Comment:
This one should be there and indicates a bug or sync problem between our model and the database. Can you remove this and we'll work on fixing the index in another ticket?

Reply:
when you say "should be there", do you mean should be in the autogenerated code, or should be in the `postgres` database?

Reply:
I mean we should add it as an index in models.py.

Reply:
It's ok to add another commit to this branch and push it up.